Detections/CommonSecurityLog/MultiVendor-PossibleDGAContacts.yaml

```diff
@@ -37,7 +37,7 @@ queryPeriod: 6h
 triggerOperator: gt
 triggerThreshold: 0
 tactics:
-  - Command and Control
+  - CommandandControl
```

Comment:
CommandAndControl case sensitive

Reply:
Done, RareClientFileAccess.yaml has been moved back to Hunting, not sure who\how it got moved, so no need for the query and trigger items for that one.